### PR TITLE
Link destination filter

### PR DESCRIPTION
**Pull request by kii-chan-reloaded**
This is a little hack I did to make the renderer only render tooltips that point to specific destinations. As an example of why this functionality might be desired, I needed this so that only links to a page in my wiki's "dictionary" namespace would render a tooltip.

### CODE REVIEW

File: conf/default.php [hunk 1]
```diff
@@ -3,3 +3,4 @@
 $conf['delay'] = 0;
 $conf['linkall_inclusions'] = '';
 $conf['linkall_exclusions'] = '';
+$conf['linkall_points_to'] = '';
```
Comment (zioth) [inline conf/default.php:6]: I'd probably call this `linkall_target_inclusions`, and create a `linkall_target_exclusions` to match.

File: renderer.php [hunk 2]
```diff
@@ -24,6 +25,13 @@ public function __construct() {
 		$this->m_exclude =
 			(!empty($inclusions) && !preg_match("/$inclusions/", $ID)) ||
 			(!empty($exclusions) && preg_match("/$exclusions/", $ID));
+
+		// Set the regex for filtering link destinations
+		$points = $this->getConf('linkall_points_to');
+		if (empty($points)) {
+			$points = ".*";
+		}
```
Comment (zioth) [inline renderer.php:33]: It would be more efficient to leave this empty, and test `empty()` in `internallink`, rather than force an extra regex test every time.

**Review by zioth — COMMENTED** [review]
I have a couple of minor suggestions, but I think this is a useful feature. I'll try to merge it in soon (with name changes, if you don't make the change yourself).

**Comment by kii-chan-reloaded** [comment]
Feel free to do as you like with it! I'm not overly familiar with PHP nor how the DokuWiki rendering system works, so I assumed what I had submitted wouldn't be the perfect solution. I had something that worked, though, so I figured I'd at least share the concept.

Thanks for all the work you've put into this plugin! It's very convenient!
